package.json

```diff
@@ -28,7 +28,7 @@
     "build-dll": "cross-env NODE_ENV=development webpack -r babel-register --config webpack.config.renderer.dev.dll.js --progress --profile --colors",
     "build-dll-live": "cross-env NODE_ENV=production webpack -r babel-register --config webpack.config.renderer.live.dev.dll.js --progress --profile --colors",
     "build-main": "cross-env NODE_ENV=production webpack -r babel-register --config webpack.config.main.prod.js --progress --profile --colors",
-    "build-auth": "cross-env NODE_ENV=production webpack -r babel-register --config app/extensions/safe/auth-web-app/webpack.config.auth-app.js --progress --profile --colors",
+    "build-auth-app": "cross-env NODE_ENV=development webpack -r babel-register --config app/extensions/safe/auth-web-app/webpack.config.js --progress --profile --colors",
```

Comment:
lets build for `production`

Reply:
We also need to add this build to the `build` command

Reply:
and automate the `yarn` for this on install too.

Reply:
@hunterlester we need this basically in a state where a fresh install will have the full auth app as needed 👍

Reply:
@joshuef I just tried with success to run `yarn` with this PR branch on a Linux VM to successfully build `auth-web-app`. And OSX was good for you, yeah?